### ISSUE TITLE
Help popup in web client not clearing after close

### ISSUE THREAD

**Issue by fang0654** [evennia/evennia#1789]
#### Brief summary of issue / Description of requested feature:

The help popup does not clear the contents when closing the window. When you type "help" again, the new help text is now at the bottom of the window, and you need to scroll down to find it. This can get a little confusing, as what is being displayed isn't necessarily what was requested.

#### Steps to reproduce the issue / Reasons for adding feature:

1. Type 'help'
2. Close the window
3. Type 'help set'
4. Observe that the help for 'set' is at the bottom of the help window.

#### Error output / Expected result of feature

Expected to clear the window between uses, so only the current help command is displayed.

**Comment by ph1234k** [comment]
I'm new to the project so I am still learning the codebase. Is it possible this is caused by the behavior of the function "evmore" which seems to be called for webclient connected sessions?

**Comment by fang0654** [comment]
Just a follow-up on this. Changing line 140 of `web/static/webclient/js/plugins/options.js` like so:

```javascript
$('#helpdialogcontent').append('<div>'+ args + '</div>');
```
to
```javascript
$('#helpdialogcontent').prepend('<div>'+ args + '</div>');
```

at least puts the new help data at the beginning, instead of at the end. This still then keeps a help history. I can put together a proper patch in a week when I get some free time again.

**Comment by friarzen** [comment]
I guess the proper solution is to just use html() instead of append():
$('#helpdialogcontent').html('<div>'+ args + '</div>');

**Comment by Griatch** [comment]
Resolved in #1887.
